src/directives/style-map.ts

```diff
@@ -82,7 +82,7 @@ export const styleMap = directive((styleInfo: StyleInfo) => (part: Part) => {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       (style as any)[name] = styleInfo[name];
     } else {
-      style.setProperty(name, styleInfo[name]);
+      style.setProperty(name, styleInfo[name] || '');
```

Comment:
We generally avoid coercing operators. I'd use a ternary, and hoist `styleInfo[name]` since it would be used 3 times then:

```ts
    previousStyleProperties.add(name);
    const value = styleInfo[name];
    if (name.indexOf('-') === -1) {
      // eslint-disable-next-line @typescript-eslint/no-explicit-any
      (style as any)[name] = value;
    } else {
      style.setProperty(name, value == null ? '' : value);
```